host/src/components/Responses/CustomLabel.jsx

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
+import React, {useCallback} from 'react';
```

Comment:
The Real Time Responses component changes when Short Answer Responses are enabled. Here is a link to the [Figma](https://www.figma.com/file/PadzU8GqyhFLCjIt7rQkpE/RightOn-Apps?type=design&node-id=15530-199518&mode=design&t=eo6L3eZ8xlSf1Cb9-4). And here's a screenshot:

<img width="401" alt="Screenshot 2023-10-31 at 8 50 30 AM" src="https://github.com/rightoneducation/righton-app/assets/79417944/d76d868c-eee5-43d6-bfab-cebbaa997d2c">

**CustomLabel** is the component that provides the answers above each of the bars ('No Response', x^2-4x-12, etc, in the image above)